src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -213,6 +223,31 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        [RavenAction("/databases/*/subscriptions/resend", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task GetSubscriptionResend()
+        {
+            var subscriptionName = GetStringQueryString("name");
+
+            using (ServerStore.Engine.ContextPool.AllocateOperationContext(out ClusterOperationContext context))
+            using (context.OpenReadTransaction())
+            await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                var subscriptionState = Database
+                    .SubscriptionStorage
+                    .GetSubscriptionFromServerStore(subscriptionName);
+
+                if (subscriptionState == null)
+                {
+                    HttpContext.Response.StatusCode = (int)HttpStatusCode.NotFound;
+                    return;
+                }
+
+                var items = SubscriptionStorage.GetResendItems(context, Database.Name, subscriptionState.SubscriptionId);
+
+                writer.WriteArray("Items", items.Select(i => i.ToJson()), context);
```

Comment:
this does not generate proper Json, we are missing start and end object here. Also most of the times this is called 'Results' ?